CHANGELOG.md

```diff
@@ -7,13 +7,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Added
+### Added basic functionality including add/delete experiment and basic filtering
```

Comment:
@yvikhlya Could you actually add this under `### Added` a la:

### Added

- Basic functionality ...

Reply:
Done